exercises/practice/robot-simulator/robot_simulator_test.gd

```diff
@@ -0,0 +1,126 @@
+	var robot_a = solution_script.new(Vector2i(0, 0), 'north')
+	robot_a.move('L')
+	var robot_b = solution_script.new(Vector2i(0, 0), 'west')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_rotating_counterclockwise_changes_west_to_south(solution_script):
+	var robot_a = solution_script.new(Vector2i(0, 0), 'west')
+	robot_a.move('L')
+	var robot_b = solution_script.new(Vector2i(0, 0), 'south')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_rotating_counterclockwise_changes_south_to_east(solution_script):
+	var robot_a = solution_script.new(Vector2i(0, 0), 'south')
+	robot_a.move('L')
+	var robot_b = solution_script.new(Vector2i(0, 0), 'east')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_rotating_counterclockwise_changes_east_to_north(solution_script):
+	var robot_a = solution_script.new(Vector2i(0, 0), 'east')
+	robot_a.move('L')
+	var robot_b = solution_script.new(Vector2i(0, 0), 'north')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_moving_forward_one_facing_north_increments_y(solution_script):
+	var robot_a = solution_script.new(Vector2i(0, 0), 'north')
+	robot_a.move('A')
+	var robot_b = solution_script.new(Vector2i(0, 1), 'north')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_moving_forward_one_facing_south_decrements_y(solution_script):
+	var robot_a = solution_script.new(Vector2i(0, 0), 'south')
+	robot_a.move('A')
+	var robot_b = solution_script.new(Vector2i(0, -1), 'south')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_moving_forward_one_facing_east_increments_x(solution_script):
+	var robot_a = solution_script.new(Vector2i(0, 0), 'east')
+	robot_a.move('A')
+	var robot_b = solution_script.new(Vector2i(1, 0), 'east')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_moving_forward_one_facing_west_decrements_x(solution_script):
+	var robot_a = solution_script.new(Vector2i(0, 0), 'west')
+	robot_a.move('A')
+	var robot_b = solution_script.new(Vector2i(-1, 0), 'west')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_moving_east_and_north_from_readme(solution_script):
+	var robot_a = solution_script.new(Vector2i(7, 3), 'north')
+	robot_a.move('RAALAL')
+	var robot_b = solution_script.new(Vector2i(9, 4), 'west')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_moving_west_and_north(solution_script):
+	var robot_a = solution_script.new(Vector2i(0, 0), 'north')
+	robot_a.move('LAAARALA')
+	var robot_b = solution_script.new(Vector2i(-4, 1), 'west')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_moving_west_and_south(solution_script):
+	var robot_a = solution_script.new(Vector2i(2, -7), 'east')
+	robot_a.move('RRAAAAALA')
+	var robot_b = solution_script.new(Vector2i(-3, 8), 'south')
+	return [true, same_field_values(robot_a, robot_b)]
+
+
+func test_moving_west_and_south(solution_script):
```

Comment:
The method name is duplicated here, so the test file is not processed correctly and therefore we don't run any tests (checked with the Actions output). I will check how much effort would it be to improve the test runner to fail in case there are no tests detected ...